data_structures/test_priority_queue.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file
+ * @author [Tuan Ho](https://github.com/ttuanho)
+ * @brief Implementation of [Priority
+ * queue](https://en.wikipedia.org/wiki/Priority_queue) an abstract data type
+ * similar to regular queue or stack data structure in which each element
+ * additionally has a "priority" associated with it.
+ * \todo Add more edge test cases:
+ * <ul>
+ *  <li> 
+ *      Handling the case where there's duplicated priority.
+ *  </li>
+ * </ul>
+ * @note
+ * @see test_queue.c
+ */
+
+#include <cassert>
+#include <iostream>
+#include <string>
+
+#include "./priority_queue.h"
```

Comment:
This shall fix `clang-tidy` warnings.
```suggestion
#include "./priority_queue.hpp"
```